html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline Elements</title>
+  </head>
+  <body>
+    <h1><b>Blue Americana Snuggie&reg; Blanket</b></h1>
```

Comment:
so when using one of the `h` tags you do not need to use `b` tag for bold text, as the `h` tags have bold text by default!